compiler/optimizer/DeadTreesElimination.cpp

```diff
@@ -480,20 +485,37 @@ int32_t TR::DeadTreesElimination::performOnBlock(TR::Block *block)
    return 0;
    }
 
+typedef std::pair<ncount_t , TR::TreeTop* > ReadBarToTreeTopMapEntry;
```

Comment:
```suggestion
typedef std::pair<const ncount_t , TR::TreeTop* > ReadBarToTreeTopMapEntry;
```

I believe this will fix the test failure.

Reply:
Thanks a lot, Leo!